modules/adding-ibmz-lpar-agent.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_with_agent_based_installer/prepare-pxe-infra-agent.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="adding-ibmz-lpar-agents_{context}"]
+= Adding {ibm-z-title} Logical Partition (LPAR) as agents
+
+Use the following procedure to manually add {ibm-z-name} agents running in an LPAR. Use this procedure only for {ibm-z-name} clusters running in an LPAR.
+
+.Procedure
+
+. Create a boot parameter file for the agents.
++
+.Sample parameter file:
+[source,yaml]
+----
+rd.neednet=1 cio_ignore=all,!condev \
+console=ttysclp0 \
+ignition.firstboot ignition.platform.id=metal
+coreos.live.rootfs_url=http://<http_server>/rhcos-<version>-live-rootfs.<architecture>.img \// <1>
+coreos.inst.persistent-kargs=console=ttysclp0
+ip=<ip>::<gateway>:<netmask>:<hostname>::none nameserver=<dns> \// <2>
+rd.znet=qeth,<network_adaptor_range>,layer2=1
+rd.<disk_type>=<adapter> \// <3>
+zfcp.allow_lun_scan=0
+ai.ip_cfg_override=1 \// <4>
+random.trust_cpu=on rd.luks.options=discard
+----
+<1> For the `coreos.live.rootfs_url` artifact, specify the matching `rootfs` artifact for the `kernel` and `initramfs` that you are starting. Only HTTP and HTTPS protocols are supported.
+<2> For the `ip` parameter, manually assign the IP address, as described in _Installing a cluster with z/VM on IBM Z and IBM LinuxONE_. 
+<3> For installations on DASD-type disks, use `rd.dasd` to specify the DASD where Red Hat Enterprise Linux CoreOS (RHCOS) is to be installed. For installations on FCP-type disks, use `rd.zfcp=<adapter>,<wwpn>,<lun>` to specify the FCP disk where RHCOS is to be installed.
```

Comment:
Consider spelling out DASD as I do not see it's full name in the overall document.

Reply:
I would leave it as is. DASD is a common term for IBM Z customers and we spelled it out once in the UPI docs in the minimum requirements.